core/Http.php

```diff
@@ -22,7 +22,7 @@ class Http
     /**
      * Returns the "best" available transport method for {@link sendHttpRequest()} calls.
      *
-     * @return string Either `'curl'`, `'fopen'` or `'socket'`.
+     * @return string|null Either curl, fopen, socket or null of no method is supported.
```

Comment:
Yes I was aware but for this example it was a bit too much? It's obvious what's happening here, the online documentation is important but just as much as when reading it in code I would say so this is a compromise